src/umpire/interface/c_fortran/wrapResourceManager.cpp

```diff
@@ -214,7 +208,7 @@ size_t umpire_resourcemanager_get_size(umpire_resourcemanager * self,
 
 umpire_allocator * umpire_resourcemanager_make_allocator_pool(
     umpire_resourcemanager * self, const char * name,
-    umpire_allocator allocator, int initial_size, int block,
+    umpire_allocator allocator, size_t initial_size, size_t block,
```

Comment:
Shouldn't these be `std::size_t`?

Reply:
Nope, because this is a C function, so no namespacing.

Reply:
Then shouldn't this file be renamed with a `.c` suffix instead of of `.cpp`?